tidy3d/components/data/data_array.py

```diff
@@ -70,21 +73,19 @@ def __init__(self, data, *args, **kwargs):
         """Initialize ``DataArray``."""
 
         # initialize with untraced data
-        data_untraced = getval(data)
-        super().__init__(data_untraced, *args, **kwargs)
-
-        # if the passed data has tracers, store them in attrs dict
         if isbox(data):
+            super().__init__(data._value, *args, **kwargs)
             self.attrs[AUTOGRAD_KEY] = data
-            # NOTE: this is done because if we pass the traced array directly, it will create a
-            # numpy array of `ArrayBox`, which is extremely slow
+        else:
```

Comment:
could also just do
```py
super().__init__(getval(data), ...)
if isbox(data):
    # set attrs
```

Reply:
Yup sounds good. I think this is left over from when I was playing around with doing different things in the init.